### PR TITLE
Add group except test for flux

### PR DESCRIPTION
**Pull request by adamperlin**
Adds an additional test case for grouping with the `except` argument. See #340.

### CODE REVIEW

**Comment by adamperlin** [comment]
Changes made.

File: query/querytest/test_cases/group_except.flux [hunk 1]
```diff
@@ -0,0 +1,4 @@
+from(db:"test")
+    |> range(start:-5m)
+    |> group(except:["_measurement"])
```
Comment (nathanielc) [inline query/querytest/test_cases/group_except.flux]: I think we want to add `_time` and `_value` to the except list otherwise every point is in its own group which makes the test hard to follow.

Comment (adamperlin) [reply query/querytest/test_cases/group_except.flux]: Agreed - done.
